_sass/custom/_faq.scss

```diff
@@ -11,7 +9,7 @@
     }
   
      h2 {
-      color: $black;
+      color: $primary;
```

Comment:
is this related to another change right? was that intended?

Reply:
@dricazenck yes, I changed the faq/accordion link titles to use our purple link color for now, to make it more clear that you can click on it.

Reply:
Now that I've reused the accordion in body content on this page

Reply:
We can also disregard this change for now until we have some accordion designs in Figma :) @dricazenck